main.go

```diff
@@ -54,16 +61,20 @@ func (collector *CollectorApp) handler(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
+	readTimeOut, _ := strconv.Atoi(os.Getenv("SERVER_READ_TIMEOUT"))
```

Comment:
We should probably check the `Getenv` result to see if it actually exists first and error out if it doesn't prior to passing it to Atoi.